doc/source/_static/code_blocks.css

```diff
@@ -0,0 +1,17 @@
+.codeblock-label{
+	display:inline-block;
+	border-top-left-radius:.5rem;
+	border-top-right-radius:.5rem;
+	font-family:Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace;
+	font-size:.75rem;--bg-opacity:1;
+	background-color:#e1e7ef;
+	background-color:rgba(224.70600000000002,231.07080000000002,239.394,var(--bg-opacity));
+	padding:.25rem .75rem;
+	border-top-width:1px;
+	border-left-width:1px;
+	border-right-width:1px;--border-opacity:1;
+	border-color:#ccd6e0;
+	border-color:rgba(204,213.85999999999999,224.39999999999998,var(--border-opacity));
+	margin-bottom:0;
+}
+
```

Comment:
"Nice" is strongly subjective! This is actually only for the `config.toml` box which is added to toml file snippets. I wanted a way to differentiate them from standard code snippets as otherwise it was confusing.

![image](https://user-images.githubusercontent.com/32061685/160402349-c896a4ed-abd1-45de-a756-ab52747585a8.png)

Might tweak this at a later date but didn't want to overly complicate for now.

Reply:
Ah I see, wonder if there's a way to leave comments in the `.css` file so we know the intended use case? Or potentially more descriptive class name?